velox/type/Filter.h

```diff
@@ -790,6 +915,42 @@ class BigintValuesUsingHashTable final : public Filter {
         values_(other.values_),
         sizeMask_(other.sizeMask_) {}
 
+  BigintValuesUsingHashTable(
```

Comment:
We can use the existing ctor and set the additional fields in `create()` since it's in the same class.
In general we probably don't need extra ctor for `create()`.